ui/appui-layout-react/src/test/state/TabLocation.test.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { should } from "chai";
+import {
+  addFloatingWidget, addPanelWidget, createNineZoneState, getTabLocation, removeTabFromWidget,
+} from "../../appui-layout-react";
+import { addWidgetState } from "../../appui-layout-react/state/internal/WidgetStateHelpers";
+import { addTabs } from "../Utils";
+
+describe("getTabLocation", () => {
+  it("should return 'undefined' if widget is not found", () => {
+    let state = createNineZoneState();
+    state = addTabs(state, ["t1"]);
+    state = addFloatingWidget(state, "w1", ["t1"]);
+    state = removeTabFromWidget(state, "t1");
+    const tab = getTabLocation(state, "t1");
+    should().not.exist(tab);
```

Comment:
I've decided to use `should` assertions (majority of assertions are `should` based in **appui-layout-react** package), since it matches the test naming more closely i.e. "should return ...".
The function return type is `Object | undefined`, so it should never return anything else than `undefined` that fulfills `should().not.exist()` assertion, unless we change the return type. I'll change the assertion to be more explicit `should().equal(tab, undefined);` in this case.